examples/standard.yaml

```diff
@@ -36,6 +36,7 @@ defaults:
         layers: 1
         mlp_hidden_dim: 512
         bridge: !CopyBridge {}
+        eps: 0.1
```

Comment:
Make this a more descriptive name, e.g. "label_smoothing".

Reply:
done via the latest commit !